e2e/specs/dataframe.spec.ts

```diff
@@ -39,7 +39,7 @@ describe("st.dataframe", () => {
 
   it("contains all numbers from 0..99", () => {
     cy.get("@cells").each(($element, index) => {
-      cy.wrap($element).should("contain", index);
+      return cy.wrap($element).should("contain", index);
```

Comment:
Was this test failing sometimes or did you add the `return` just in case even though there is no snapshot?

Reply:
I'm not aware of it failing, but as all checks are promises it does not hurt to add the `return` on every `each`